src/app/commons/ItemListPage/ItemListPage.js

```diff
@@ -27,12 +27,19 @@ const FilterSearchBar = ({filterItems, Button}) => {
  *  receives an index of the data and then return the content (html) of that within the <td></td>
  *  (3) key: a function that determines the key of the item given a student
  * @param tableDataStyle the custom style of the <td> elements
+ * @param showFilterSearchBar the decision (boolean) of showing default <FilterSearchBar> component
+ * @param draggable the decision (boolean) of <tr> draggable elements
+ * @param ondrag the operation (obj) of <tr> elements dragging
+ * @param ondrop the operation (obj) of <tr> elements dropping
```

Comment:
Please help correct this to the JsDoc writing format.

You can reference `ConfirmationModal`'s JsDoc